pkg/provider/aws/resources/region.go

```diff
@@ -43,7 +43,7 @@ func (region *Region) Id() core.ResourceId {
 	return core.ResourceId{
 		Provider: AWS_PROVIDER,
 		Type:     REGION_TYPE,
-		Name:     region.Name,
+		Name:     "region",
```

Comment:
Do we want to make these const somewhere?

Reply:
yeah i can make these constant, it will be easier to reference throughout